src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -211,8 +211,8 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             new object[] { Configuration.Http.WrongHostNameCertRemoteServer , SslPolicyErrors.RemoteCertificateNameMismatch},
         };
 
-        [ActiveIssue(7812, TestPlatforms.Windows)]
-        [OuterLoop] // TODO: Issue #11345
+        // [ActiveIssue(7812, TestPlatforms.Windows)]
+        // [OuterLoop] // TODO: Issue #11345
```

Comment:
Whether or not it's marked Outerloop, I'd suggest doing outerloop runs, so that all of the other outerloop tests also run.  Whatever's behind this issue, it's influenced by some kind of race condition or other external factors.